docs/options/config-file.md

```diff
@@ -66,13 +66,12 @@ This is useful if you want to rely solely on command-line options or default set
 
 `tsdown` supports multiple config loaders to accommodate various file formats. You can select a config loader using the `--config-loader` option. The available loaders are:
 
-- `auto` (default): Utilizes native runtime loading for TypeScript if supported; otherwise, defaults to `unconfig`.
+- `auto` (default): Utilizes native runtime loading for TypeScript if supported; otherwise, defaults to `unrun`.
 - `native`: Loads TypeScript configuration files using native runtime support. Requires a compatible environment, such as the latest Node.js, Deno, or Bun.
-- `unconfig`: Loads configuration files with the `unconfig` library, ensuring broad compatibility across different runtimes.
-- `unrun` _(experimental)_: Loads configuration files using the [`unrun`](https://gugustinette.github.io/unrun/) library, which provides similar compatibility as `unconfig` but with more performances. Install the `unrun` package to use this loader.
+- `unrun`: Loads configuration files using the [`unrun`](https://gugustinette.github.io/unrun/) library. Install the `unrun` package to use this loader.
```

Comment:
> Install the `unrun` package to use this loader.

it's now installed by default as direct runtime dependency.
which reverts the fix made in #570 , @sxzz

Reply:
Yes. I will update the docs.

Reply:
tsdown and unrun will ensure that the rolldown versions remain consistent before release, up until rolldown officially launches its stable version.

Reply:
I understand, my point is that if I'm using `.ts` extensions in imports so that don't even need `unrun`, it's still installed anyway @sxzz